yt_dlp/extractor/la7.py

```diff
@@ -74,9 +79,7 @@ def _real_extract(self, url):
         formats.extend(m3u8_formats)
 
         for q in filter(None, video_path.split(',')):
-            http_f = self._generate_mp4_url(q, m3u8_formats)
-            if http_f:
-                formats.append(http_f)
+            formats.append(self._generate_mp4_url(q, m3u8_formats))
```

Comment:
There must have been some reason for the `if`. Don't remove it unless u r sure

Reply:
yes, you are correct. Adding none to the formats list creates an error.